packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -181,6 +182,21 @@ abstract class PatrolCommand extends Command<int> {
     return argResults![name]! as List<String>? ?? <String>[];
   }
 
+  FlutterCommand get flutterCommand {
+    final arg = globalResults!['flutter-command'] as String?;
+
+    var cmd = arg;
+    if (cmd == null || cmd.isEmpty) {
```

Comment:
Have you tested the scenario with defining flutter command as environment variable? In `patrol_command_runner.dart` `flutter-command` defaults to `flutter`. I think it will cause that this condition will always be false.

Reply:
Good catch, sorry about that! That's fixed now:

![grafik](https://github.com/leancodepl/patrol/assets/80046268/c2786117-247d-4002-be82-69df6632b6d6)